### ISSUE TITLE
Focus lost after checking or unchecking "Enhanced Sound" from PhET Menu

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#219]
@jbphet, I noticed that when checking Enhanced Sound in the PhET Menu, the pop-up menu closes and focus is lost.

I noticed this in GFL and in John Travoltage, so i think this is a general issue.

Keyboard focus should be returned to the PhET Menu button when the popped up menu closes.

For an example try accessing the About page and then closing the About page dialog. You'll see that focus returns to the PhET menu button.

**Comment by terracoda** [comment]
This is happening on master and MacOS and Safari. I didn't test anything else.

**Comment by jessegreenberg** [comment]
This is related to keyboard navigation so assigning to myself.

**Comment by jessegreenberg** [comment]
This is an issue for all menu items that don't open a dialog. We have logic to move focus from menu to dialog, and dialog back to menu button. But not back to menu button after selection. This applies to screenshot, full screen, and enhanced sounds buttons.

**Comment by jessegreenberg** [comment]
Discussed with @zepumph -

The current option we have for this (focusAfterCallback) is not general enough. It optionally moves focus to top of the document after the menu item callback. We are going to replace it with an optional function so that you can specify focus behavior. Maybe called `handleFocusCallback`. We considered not using this option and instead going right through the MenuItem `callback` option. But then we would need on copy of `if ( event.pointer.isA11y() ) {...}` for each instance of MenuItem.

**Comment by jessegreenberg** [comment]
Done in the above commits.

**Comment by jessegreenberg** [comment]
@terracoda can you please confirm this is fixed in https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-dev.22/phet/gravity-force-lab_en_phet.html?a11y&supportsSound&supportsEnhancedSound?

**Comment by terracoda** [comment]
@jessegreenberg, the focus is working great now, thanks.
I'll message you on slack about a separate issue and where to put.
Closing this one.
